src/Domain/Structure/CreateInitialMesh.hpp

```diff
@@ -24,19 +26,29 @@ namespace domain::Initialization {
 /// \ingroup InitializationGroup
 /// \brief Construct the initial Mesh of an Element.
 ///
-/// \details When constructing the Mesh of an Element, pass its id, and use the
-/// default argument for orientation.  When constructing the mesh of a
-/// neighboring Element (when constructing mortars), pass the id and orientation
-/// of the neighbor.
+/// \param initial_extents initial extents for Elements in each Block of the
+///        Domain
+/// \param element Element
+/// \param legendre_quadrature the quadrature rule/grid point distribution for
+///        dimensions that use Spectral::Basis::Legendre
+template <size_t Dim>
+Mesh<Dim> create_initial_mesh(
+    const std::vector<std::array<size_t, Dim>>& initial_extents,
+    const Element<Dim>& element, Spectral::Quadrature legendre_quadrature);
```

Comment:
Just one overload that takes the topologies array is enough I think? Plus initial extents, element id, quadrature as before. Then also you don't need to construct full elements/blocks for testing.

Reply:
I considered this, but am worried that some user will pass in the block topologies instead of the element topologies which will be incorrect once refining B2Radial is supported

Reply:
Ok